src/components/DevTools/DevTools.wallet.tsx

```diff
@@ -35,7 +35,7 @@ class BsIdentity {
 
   public static async create(alias: string): Promise<void | IMyIdentity> {
```

Comment:
Is this still async now?

Reply:
No, I altered the `BsIdentity.save` so both are no longer async.